.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,13 +27,24 @@ Optionally add one or more of the following kinds if applicable:
 
 #### What this PR does / why we need it:
 
-#### Which issue(s) this PR fixes:
+#### Which issue(s) this PR is related to:
+
```

Comment:
Nit: Let's keep it one line shorter by not inserting this newline.

Reply:
done!